pkg/services/accesscontrol/resolvers.go

```diff
@@ -16,7 +16,9 @@ type ScopeAttributeResolver interface {
 }
 
 type ActionResolver interface {
-	Resolve(action string) []string
+	ResolveAction(action string) []string
+	ResolveActionSet(actionSet string) []string
+	ExpandActions(permissions []Permission) []Permission
```

Comment:
```suggestion
	ExpandActions(permissions []Permission) []Permission
```

so we get permissions and get permissions out?

is it not?
```go
	ExpandActions(actionSets []string) []Permission
```

Reply:
yea, apologize. i should come up with a solution as well.

perhaps we could use

```go
ExpandActionSetsToActions
```

```go
ExpandActionSets
```

Reply:
nit: we can always change this back if it's just about naming convention. I'd prefer to be a bit more descriptive to `ExpandingActionSetsToActions` , to indicate what is happening

Maybe even using something like this (toooo  long, but to get the description of what is happening):

```
ExtendPermissionsWithActionSetsToActions
```